pkg/crc/machine/machine.go

```diff
@@ -190,6 +190,18 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		logging.WarnF("Failed Public dns query: %v : %s", err, queryOutput)
 	}
 
+	// Copy Kubeconfig file from bundle extract path to machine directory.
+	// In our case it would be ~/machine/crc
+	logging.InfoF("Copying kubeconfig file to instance dir ...")
+	if err := crcos.CopyFileContents(
+		filepath.Join(extractedPath, "kubeconfig"),
+		filepath.Join(constants.MachineInstanceDir, machineConfig.Name, "kubeconfig"),
+		0644); err != nil {
+		logging.ErrorF("Error to copy kubeconfig content %v", err)
```

Comment:
You could change this error message to: `Error while copying kubeconfig content`